docs/src/reference/gremlin-variants.asciidoc

```diff
@@ -304,7 +304,7 @@ The following table describes the various configuration options for the Gremlin
 |connectionPool.maxInProcessPerConnection |The maximum number of in-flight requests that can occur on a connection. |4
 |connectionPool.maxSimultaneousUsagePerConnection |The maximum number of times that a connection can be borrowed from the pool simultaneously. |16
 |connectionPool.maxSize |The maximum size of a connection pool for a host. |8
-|connectionPool.maxWaitForConnection |The amount of time in milliseconds to wait for a new connection before timing out. |3000
+|connectionPool.maxWaitForConnection |The amount of time in milliseconds to wait for a new connection before timing out. |25000
```

Comment:
Do these new numbers have a reference, or are they just arbitrary?

Reply:
Arbitrary. However, the default SSL handshake timeout is too low for some services so increasing that to 15000 (from 10000) forced me to increase the other values too since they all need to be larger than one another.